redux-logger/redux-logger.d.ts

```diff
@@ -8,13 +8,15 @@
 declare module 'redux-logger' {
 
   interface ReduxLoggerOptions {
+    actionTransformer?: (action: any) => any;
     collapsed?: boolean;
+    duration?: boolean;
     level?: string;
     logger?: any;
+    predicate?: (getState: Function, action: any) => boolean;
     timestamp?: boolean;
-    transformer?: (state:any)=>any;
-    predicate?: (getState:Function, action:any)=>boolean;
+    transformer?: (state:any) => any;
```

Comment:
As actions state must be type "any", because the state can contain any of the properties.